### ISSUE TITLE
No path to be recognized for your work in SIGs that operate outside of GitHub

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/sig-contributor-experience#11]
There are multiple SIGs (especially in SemConv) that operate a lot in google docs & slack, this means that work of individual contributors is not tracked on CNCF DevStats, which for example is used for checking who can vote for GC. Also this data is relevant for members applying to the github org, etc.

From the original meeting notes: Semantic Conventions is a mono-repo but has multiple SIGs contributing to it,
it is confusing, how it works, how do you end up being an approver, etc? Work is limited to slack and google docs. No way to collaborate as a group outside of that (as far as I know)
* Experiments of those semantics can be very very long time
* No path to be “recognized” for your work until stuff is materialized 
* Similar issues with code owners in collector (and language contrib repos)
* We can not tell SIGs how to operate, but we can consult/support (similar to End User SIG does)
* Do we need a “labs area”, so contributions can materilize earlier (people can start their journey earlier, their contributions get recognized earlier)
  * https://github.com/open-telemetry/oteps/issues/231

**Comment by codefromthecrypt** [comment]
TL;DR; If we had some insight into what it would take to bring in more signals, specifically cost, Google Docs/Slack could stop being an immutable unsolvable community signal.

Regardless of edge cases, I think it is a norm of SIGs to use google docs (not always viable either for folks in china) and slack (honestly not sure if this is an issue for folks in china).

One earnest question is to what degree folks in devstats are interested in multiple signals. It is certainly easier to look at one source of data, and even with that there is bookkeeping about which authors are which people and which companies. I do wonder if raking stats from google docs or slack have been explored. As both google and slack are CNCF regulars, it seems someone could ask them :)

Any viability on this could help us at least answer why. For example, it is too expensive and CNCF doesn't have enough funds for contractor work to do this, or to pay for a data feed. If we get to the bottom of it, it could become solved.

**Comment by svrnm** [comment]
> TL;DR; If we had some insight into what it would take to bring in more signals, specifically cost, Google Docs/Slack could stop being an immutable unsolvable community signal.

This may work for slack, but google docs and especially attending zoom meetings is fairly impossible to track. Is "adding a line to a google docs" 1 contribution? Is "sitting in a zoom meeting" 1 contribution?. The github-based tracking is already far from perfect, but it gives some indiciation on how people contribute. 

Eventually we need to consider a more human-to-human solution, established members of the community (especially maintainers) should be encouraged to highlight the work of others and help them to go through the "contributor ladder", e.g. I run into contributors again and again that do not know how to become a member of the GH org, or that this is how we signal "you are an established member". We might also need to consider to review what it means to be "an approver" in non-repository-based SIGs, or how we can make non-repository-based SIGs into ones that have a repo (see the End User SIG example once again). Or, to put it differently: instead of writing things in google docs, they should be written in github (issues, or PRs), as often as possible.
